### PR TITLE
Collapsable groups

### PR DESCRIPTION
**Pull request by Guntrisoft**
Create a concept of collapsable groups which fire an event when a group item is clicked so that the others can close up if required.

This turns the collapsables in the group into an 'accordion'

### CODE REVIEW

**Comment by Guntrisoft** [comment]
I need to write some tests for this, so created this PR a bit prematurely!

**Comment by ourmaninamsterdam** [comment]
[WIP] it :+1:

File: assets/js/components/Collapsable.js [hunk 1]
```diff
@@ -96,9 +103,21 @@ define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, Dough
...
     this.$triggers[isActive ? 'on' : ' off']('click', $.proxy(function() {
+      if (this.config.oneGroupOpenOnly) {
+        eventsWithPromises.publish('toggler:close:' + this.groupName, this.$el);
+      }
+
       this.toggle();
     }, this));
 
+    if (this.config.oneGroupOpenOnly) {
+      eventsWithPromises.subscribe('toggler:close:' + this.groupName, $.proxy(function(el) {
```
Comment (ourmaninamsterdam) [inline assets/js/components/Collapsable.js]: I think `el` would be more descriptive as `$el`.

**Comment by ourmaninamsterdam** [comment]
:+1:

**Comment by benbarnett** [comment]
:+1:

**Pull request merged**